### ISSUE TITLE
Add a keyboard shortcut to jump to end of EPUB

### ISSUE THREAD

**Issue by PhilippeBruno** [edrlab/thorium-reader#1125]
Similar to ADE, a shortcut to quickly jump to end of EPUB (often to look up the copyright information... a must when citing a book) would be a welcome feature.

The keyboard shortcut should be user configurable like the other shortcuts, but should default to [Ctrl][End] on Windows (perhaps [Command][End] on macOS... I am not familiar with this environment).

**Comment by danielweck** [comment]
This was implemented. Wiki doc https://github.com/edrlab/thorium-reader/wiki/Keyboard-shortcuts
